src/sentry/tagstore/snuba/backend.py

```diff
@@ -16,69 +17,363 @@
 from sentry.tagstore.base import TagStorage
 from sentry.utils import snuba
 
+SEEN_COLUMN = 'received'
+
+
+class ObjectWrapper(object):
+    def __init__(self, dictionary):
+        self.__dict__ = dictionary
+
 
 class SnubaTagStorage(TagStorage):
 
+    def get_time_range(self, days=90):
+        """
+        Returns the default (start, end) time range for querrying snuba.
+        """
+        end = datetime.utcnow().replace(tzinfo=pytz.UTC)
```

Comment:
nit: I noticed in search we seem to use `django.utils.timezone.now` instead of `datetime.utcnow().replace(tzinfo=pytz.UTC)`. It apparently does the same thing but respects some Django setting?